definitions/npm/commander_v2.x.x/flow_>=0.28.x/commander_v2.x.x.js

```diff
@@ -0,0 +1,276 @@
+declare class $npm$commander$Command extends events$EventEmitter {
+  /**
+   * Initialize a new `Command`.
+   *
+   * @param {String} name
+   * @api public
+   */
+  constructor(name?: string): $npm$commander$Command;
+
+  /**
+   * Add command `name`.
+   *
+   * The `.action()` callback is invoked when the
+   * command `name` is specified via __ARGV__,
+   * and the remaining arguments are applied to the
+   * function for access.
+   *
+   * When the `name` is "*" an un-matched command
+   * will be passed as the first arg, followed by
+   * the rest of __ARGV__ remaining.
+   *
+   * Examples:
+   *
+   *      program
+   *        .version('0.0.1')
+   *        .option('-C, --chdir <path>', 'change the working directory')
+   *        .option('-c, --config <path>', 'set config path. defaults to ./deploy.conf')
+   *        .option('-T, --no-tests', 'ignore test hook')
+   *
+   *      program
+   *        .command('setup')
+   *        .description('run remote setup commands')
+   *        .action(function(){
+   *          console.log('setup');
+   *        });
+   *
+   *      program
+   *        .command('exec <cmd>')
+   *        .description('run the given remote command')
+   *        .action(function(cmd){
+   *          console.log('exec "%s"', cmd);
+   *        });
+   *
+   *      program
+   *        .command('*')
+   *        .description('deploy the given env')
+   *        .action(function(env){
+   *          console.log('deploying "%s"', env);
+   *        });
+   *
+   *      program.parse(process.argv);
+   *
+   * @param {String} name
+   * @param {String} [desc]
+   * @param {Mixed} [opts]
+   * @return {Command} the new command
+   * @api public
+   */
+  command(name: string, desc?: string, opts?: any): $npm$commander$Command;
+
+  /**
+   * Parse expected `args`.
+   *
+   * For example `["[type]"]` becomes `[{ required: false, name: 'type' }]`.
+   *
+   * @param {Array} args
+   * @return {Command} for chaining
+   * @api public
+   */
+  parseExpectedArgs(args: Array<string>): $npm$commander$Command;
+
+  /**
+   * Register callback `fn` for the command.
+   *
+   * Examples:
+   *
+   *      program
+   *        .command('help')
+   *        .description('display verbose help')
+   *        .action(function(){
+   *           // output help here
+   *        });
+   *
+   * @param {Function} fn
+   * @return {Command} for chaining
+   * @api public
+   */
+  action(fn: (...args: any[]) => void): $npm$commander$Command;
```

Comment:
Probably best to specify the return type of the callback as `mixed`.
This lets people use general-purpose functions (that might happen to return something) as callbacks.